skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -52,11 +53,26 @@ class SkaffoldDevConfiguration(
     project: Project,
     factory: ConfigurationFactory,
     name: String
-) : RunConfigurationBase(project, factory, name) {
+) : BaseSkaffoldRunConfiguration(project, factory, name) {
 
     override fun getState(executor: Executor, environment: ExecutionEnvironment): RunProfileState? =
         null
 
     override fun getConfigurationEditor(): SettingsEditor<out RunConfiguration> =
         SkaffoldDevSettingsEditor()
 }
+
+/**
+ * Base class for Skaffold run configurations, includes base properties such as Skaffold
+ * configuration file path.
+ */
+abstract class BaseSkaffoldRunConfiguration(
```

Comment:
Instead of `BaseSkaffoldRunConfiguration`, how about just `SkaffoldRunConfiguration`?

Reply:
I agree `Base` maybe a bit misleading, but just saying `SkaffoldRunConfiguration` may imply this configuration is used as is which is not quite true, since we intend to keep common settings here. How about `CommonSkaffoldRunConfiguration`? or maybe `Generic`

Reply:
or we can just be blunt about it and say `AbstractSkaffoldRunConfiguration`?

Reply:
Went with just `SkaffoldRunConfiguration`, after giving it another thought, it should be good enough.

Reply:
Although it is indeed abstract, let's do `AbstractSkaffoldRunConfiguration` instead if you don't object.